cs-multi-shell/package.json

```diff
@@ -33,6 +33,8 @@
     "@capacitor/keyboard": "6.0.1",
     "@capacitor/network": "6.0.1",
     "@capacitor/status-bar": "6.0.0",
+    "@ionic-enterprise/auth": "6.0.0",
```

Comment:
Auth Connect and Identity Vault are npm installed. See the install instructions. You will need to run `ionic enterprise register` to add your product key.